.github/workflows/push_deploy.yml

```diff
@@ -1,29 +1,37 @@
 name: Push Deploy
 
 on:
-  workflow_dispatch:
-  push:
-    branches:
-      - master
-    paths:
-      - 'Sources/PushServer/**'
+  release:
+    types:
+      - published
```

Comment:
probably not going to be able to do this repo's version numbers to this push server. the downsides of this 'monorepo'. i think it's gotta be these existing -- push to master or workflow dispatch

Reply:
Why?

Reply:
This does not change that?

Reply:
It would require we tag more often than we release app updates, and the 'latest' release would no longer be guaranteed to have an attached binary.